### PR TITLE
Define and activate AWS 9.2.0 legacy release

### PR DESCRIPTION
**Pull request by sslavic**

### CODE REVIEW

**Comment by sslavic** [comment]
Upgrading from 9.1.0 to 9.2.0, and clean 9.2.0 install have been tested in giraffe (AWS, China)

**Comment by sslavic** [comment]
Upgrading from 9.1.0 to 9.2.0, and clean 9.2.0 install have been tested on ginger (AWS, EU)

**Comment by xh3b4sd** [comment]
For my interest, why is this a minor bump? Why not major or patch?

**Comment by sslavic** [comment]
nginx default ssl ciphers have changed, dropping support for some older browsers; we've made appropriate note in release notes, still we want to bring attention to this user affecting change in additional way through making this minor bump.

File: aws.yaml [hunk 1]
```diff
@@ -146,6 +146,44 @@
...
+      componentVersion: 0.29.0
+      version: 1.5.0
+    - app: node-exporter
+      componentVersion: 0.18.1
+      version: 1.2.0
+  authorities:
+    - name: app-operator
+      version: 1.0.0
+    - name: aws-operator
+      version: 5.6.0
```
Comment (pipo02mix) [inline aws.yaml]: Why chart operator is not listed in this last one as authority?

Comment (sslavic) [reply aws.yaml]: @rossf7 explained in https://github.com/giantswarm/releases/pull/84#discussion_r381200033

If I'm not mistaken, same rationale applies for legacy aws releases like this too.

**Pull request merged**